dist/aurora.js

```diff
@@ -1,6 +1,23 @@
 //[aurora]
 // Uncomment the previous line for testing on webpagetest.org
 
+// Wrap individual metric calls so errors don't break metrics in entire file.
+function runSafely(cb) {
+  try {
+    return cb() || null;
+  } catch (e) {
+    return logError(e);
+  }
+}
+
+function logError(e) {
+  try {
+    return `${e?.name}: ${e?.message}`;
```

Comment:
A concrete prefix to anchor on, eg in case you need to filter out pages with errors.

```suggestion
    return `Error ${e?.name}: ${e?.message}`;
```

Reply:
Makes sense, committed your suggestion!